src/js/_enqueues/lib/comment-reply.js

```diff
@@ -189,9 +189,10 @@ window.addComment = ( function( window ) {
 			parentId  = getDataAttribute( replyLink, 'commentid' ),
 			respondId = getDataAttribute( replyLink, 'respondelement'),
 			postId    = getDataAttribute( replyLink, 'postid'),
+			replyTo   = getDataAttribute( replyLink, 'replyto'),
 			follow;
 
-		if ( ! commId || ! parentId || ! respondId || ! postId ) {
+		if ( ! commId || ! parentId || ! respondId || ! postId || ! replyTo ) {
```

Comment:
This was for back-compat with sites that had a custom comments walker using inline JS, ie `onclick=moveForm( /* etc */ );`.

To allow for sites that have a custom walker with the new data attributes, it would be good to set a default `replyTo` if the attribute isn't set rather than require it.

Reply:
Hmm, the default `replyTo` string will have to be translatable, won't it? How can I import `__()` to use in here?